tasks/build.js

```diff
@@ -6,5 +6,6 @@ gulp.task('build', gulp.series(
   'build:test',
   'build:website',
   'build:boilerplates',
-  'copy:chi:ce-scripts'
+  'copy:chi:ce-scripts',
+  'copy:chi:vue-scripts'
```

Comment:
This does not seem to be working as expected. I ran `chi build` and `/dist/js/vue` folder is empty.

Reply:
`chi test` and `chi build` create dist folder in `/src/chi-vue` and that's where the scripts are.
The folder must be cleaned, otherwise, the next `chi start`, `chi build` or `chi test` throws error
src/chi-vue/dist is a directory. Please, remove it